ext/nnstreamer/extra/nnstreamer_grpc_common.h

```diff
@@ -59,6 +59,16 @@ class NNStreamerRPC {
       return handle_;
     }
 
+    /** @brief check it's server instance */
+    gboolean getIsServer () {
```

Comment:
minor comment, suggestion getIsServer > isServer

Reply:
will be updated.